include/deepstream/core/fwd.hpp

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2017 deepstreamHub GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef DEEPSTREAM_CORE_FWD_HPP
+#define DEEPSTREAM_CORE_FWD_HPP
+
+namespace deepstream {
```

Comment:
Previously the declarations in this file were listed at the top of all the other header files (or the ones that needed to reference them). This just groups them together, reducing the burden of adding more, but increasing the header file count slightly.